--- a/src/atvclient.cpp	2015-08-25 22:52:31.730703980 +0400
+++ b/src/atvclient.cpp	2015-08-25 22:54:35.656806402 +0400
@@ -172,7 +172,7 @@
 static CPacketNOTIFICATION remote_unpaired("Remote unpaired", "You can now control XBMC with any Apple remote.", NULL, NULL);
 static CPacketNOTIFICATION remote_pair_failed("Remote already paired", "This AppleTV was paired to another remote. To unpair, hold down menu and rewind for 6 seconds.", NULL, NULL);
 
-const char* remoteIdFile = "/etc/atvremoteid";
+const char* remoteIdFile = "/storage/.cache/atvremoteid";
 static int pairedRemoteId = 0;
 
 // new globals to support NEC mode
