--- a/xbmc/windows/GUIWindowSystemInfo.cpp	2015-07-13 11:19:18.000000000 +0400
+++ b/xbmc/windows/GUIWindowSystemInfo.cpp	2015-07-25 09:50:37.621917837 +0400
@@ -56,7 +56,7 @@
   case GUI_MSG_WINDOW_INIT:
     {
       CGUIWindow::OnMessage(message);
-      SET_CONTROL_LABEL(52, CSysInfo::GetAppName() + " " + g_infoManager.GetLabel(SYSTEM_BUILD_VERSION).c_str() +
+      SET_CONTROL_LABEL(52, "AppleTV running " + CSysInfo::GetAppName() + " " + g_infoManager.GetLabel(SYSTEM_BUILD_VERSION).c_str() +
                             " (Compiled: " + g_infoManager.GetLabel(SYSTEM_BUILD_DATE).c_str() +")");
       CONTROL_ENABLE_ON_CONDITION(CONTROL_BT_PVR,
                                   PVR::CPVRManager::Get().IsStarted());
